ci-operator/config/openshift-qe/ocp-qe-perfscale-ci/openshift-qe-ocp-qe-perfscale-ci-main__aws-4.15-nightly-x86.yaml

```diff
@@ -50,6 +50,18 @@ tests:
     - chain: ipi-aws-pre
     - chain: create-infra-move-ingress-monitoring-registry
     workflow: openshift-qe-control-plane
+- as: control-plane-24nodes-openshiftsdn
+  cron: 0 15 * * 2
+  steps:
+    cluster_profile: aws-perfscale-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      COMPUTE_NODE_REPLICAS: "24"
+      NETWORK_TYPE: OpenShiftSDN
+    post:
+    - chain: ipi-aws-post
+    pre:
+    - chain: ipi-aws-pre
```

Comment:
https://github.com/openshift/release/blob/master/ci-operator/step-registry/ipi/aws/sdn/pre/ipi-aws-sdn-pre-chain.yaml 

Should we just call this ?

Reply:
Yeah right, we can do the same stuff via this https://github.com/openshift/release/blob/master/ci-operator/step-registry/sdn/conf/sdn-conf-commands.sh#L6